src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -111,6 +111,12 @@ sentry {
     // you don't need to do it manually.
     // Default is disabled.
     uploadNativeSymbols = true
+
+    // Does or doesn't include the source code of native code for Sentry.
+    // This executes sentry-cli with the --include-sources param. automatically so
```

Comment:
`h`: what happens if only `includeNativeSources` is set, but not `uploadNativeSymbols`? need to point out if this setting requires the above to be set also

Reply:
Good point, this can be fixed here https://docs.sentry.io/platforms/android/gradle/ as well

Reply:
@marandaneto Yeah, make sense, I've copied it from there.